### DISCUSSION TITLE
Cannot re-register user after delete.

### DISCUSSION THREAD

**Discussion by indrazm** [pocketbase/pocketbase#3276] [Q&A]
Hey i am currently developing something with PB as backend. 
I am trying the register user test. 

After i do register, it works.
Then i delete the user. 

When i do re-register with the same data,
It returns old data, but i see nothing on the users table. 

Any Advice ?
<img width="370" alt="image" src="https://github.com/pocketbase/pocketbase/assets/114550938/8a6d711a-1fba-4def-b95f-78727ccbc5e6">
<img width="1254" alt="image" src="https://github.com/pocketbase/pocketbase/assets/114550938/42e477b2-d409-4e27-8bec-5b6152939a95">

**Comment by ganigeorgiev** [comment]
Double check that you are sending to the correct url.

It could be also a cache issue, I don't know.
If you use a Next.js or a reverse proxy (nginx, apache, caddy, etc.) check their configurations.

To rule out PocketBase issue, try to send direct API request with curl or any other http client OR try to reproduce it by creating the same user from the Admin UI.
